app/models/api/hses.rb

```diff
@@ -16,21 +16,35 @@ def error_type
   def host
     Rails.configuration.x.hses.api_hostname
   end
+
+  def request
+    Rails.logger.debug <<~EODM
+      HSES #{path}:
+      #{response.inspect}
+    EODM
+    if response.failed?
+      Rails.logger.error "HSES call to #{path} responded with #{response.code}"
+    end
+    response
+  end
 end
 
 class Api::Hses::Issue < ApiRequest
   include Api::Hses
-  include FakeApiResponseWrapper
   attr_accessor :id
 
   def initialize(id:)
     @id = id
   end
 
-  # TODO update this with real data once we have /issue endpoint
-  def request
-    fake_issue = FakeIssues.instance.data.find { |c| c[:id] == id }
-    details_response(fake_issue || FakeIssues.instance.data.last)
+  private
+
+  def path
+    "/issues-ws/issue/#{id}"
```

Comment:
[thought] should we be using some sort of URL builder here instead of string interpolation? I'm trying to think of some sort of `id` that would result in really weird behavior, and/or a security hole.

Reply:
Hmm.  You want like, `File.join("directory", "file")` but for URLs?  This whole thing ultimately gets run through `URI::HTTPS.build(uri_params)` but I guess I don't know what kind of safeguards that has.

Reply:
i think it has to be an integer, so we could add a check -- in terms of a builder, can you tell me more about what you might be envisioning? this is just copying what the other similar path methods are doing, so i didn't think about it much

Reply:
> This whole thing ultimately gets run through `URI::HTTPS.build(uri_params)`

good point! Some quick testing reveals that the most obvious weird thing to include, a space, results in an `URI::InvalidComponentError` so it's at least not blindly passing input through. This is probably fine

Reply:
> can you tell me more about what you might be envisioning

Jess nailed what I was thinking, and also why it's not needed